tests/certification/state/cockroachdb/cockroachdb_test.go

```diff
@@ -156,14 +156,14 @@ func TestCockroach(t *testing.T) {
 			Value: "v3",
 			ETag:  &etag100,
 		})
-		assert.Equal(t, fmt.Errorf("no item was updated"), err)
+		assert.Equal(t, state.NewETagError(state.ETagMismatch, nil), err)
```

Comment:
@philliphoff FYI this proves that the correct error is now returned -- cockroachdb uses the postgres implementation for this.